README.md

```diff
@@ -90,6 +90,8 @@ Choose this option to build on platforms other than linux/amd64 and/or not have
 
 The resulting image will be tagged as `grafana/grafana:dev`
 
+Notice: If you are using Docker for MacOS, be sure to let limit of Memory bigger than 2 GiB, otherwize you may faild at `grunt build`
```

Comment:
Would be nice to include how to set the memory limit as well.

Reply:
I will add it as soon as possible